app/src/main/res/values/dimens.xml

```diff
@@ -605,4 +605,8 @@
   <dimen name="topic_lessons_title_margin_start">32dp</dimen>
   <dimen name="topic_lessons_title_margin_top">32dp</dimen>
   <dimen name="topic_lessons_title_margin_end">32dp</dimen>
+
+  <!-- StoryTextSize -->
```

Comment:
Update to `Options story text size` as this is not a file instead just a regular comment. For files we use `AbcXyz`